### DISCUSSION TITLE
What are the advantages and disadvantages of each of IVF_HNSW and HNSW?

### DISCUSSION THREAD

**Discussion by freefoxcm** [milvus-io/milvus#18435] [Q&A and General discussion]
What are the advantages and disadvantages of each of IVF_HNSW and HNSW? What kind of application scenarios are they suitable for?

**Comment by yhmo** [comment]
The IVF_HNSW is experimental and was removed from v2.1.
Properties of each index type are listed in this page: https://milvus.io/docs/v2.0.x/index.md
The HNSW is high-speed, high recall rate, but requires large memory resource.
